tests/functional/test_install.py

```diff
@@ -271,6 +271,7 @@ def test_install_from_local_directory(script, data):
     assert egg_info_folder in result.files_created, str(result)
 
 
+@pytest.mark.skipif("sys.version_info[:2] == (3, 3)")
```

Comment:
^^
In fact, I'm wondering if it is worth having two distinct tests or if switching to a single with `-qqq` would make more sense.